Foundation/include/Poco/FileStream.h

```diff
@@ -79,7 +79,7 @@ class Foundation_API FileIOS: public virtual std::ios
 	NativeHandle nativeHandle() const;
 		/// Returns native file descriptor handle
 
-	Poco::UInt64 size() const;
+	UIntPtr size() const;
```

Comment:
I don't think that changing the return type is appropriate.

Reply:
I described it below.
The main idea - that filesize is platform depended

Reply:
I have reverted types